data/json/items/gun/44paper.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "colt_army",
+    "type": "GUN",
```

Comment:
Seems overriding is easier and, in that case, saves on text. I am assuming the revolver template has a default reload of 70? What about dispersion?